### PR TITLE
fix 404

### PR DESCRIPTION
**Pull request by zhouyun-zoe**
Fix 404 in docs

### CODE REVIEW

File: docs/cita/rpc-guide/rpc.md [hunk 1]
```diff
@@ -770,7 +770,7 @@ Result:
 
 Example
 
-以`scripts/contracts/tests/contracts/SimpleStorage.sol`这个合约为例子，正常在链上创建该合约；
+以`https://github.com/cryptape/test-contracts/blob/a7b95c771722f8bd4f3db5697b388c06434067f2/SimpleStorage.sol`这个合约为例子，正常在链上创建该合约；
```
Comment (kaikai1024) [inline docs/cita/rpc-guide/rpc.md]: 这个怎么指定了 commit

Comment (zhouyun-zoe) [reply docs/cita/rpc-guide/rpc.md]: Done，I change it to `https://github.com/cryptape/test-contracts/blob/master/SimpleStorage.sol`

**Review by kaikai1024 — DISMISSED** [review]

**Comment by kaikai1024** [comment]
A huge pr...

**Comment by boundless-forest** [comment]
Merge firstly, then review from website url. This way ok?

**Review by kaikai1024 — APPROVED** [review]

**Review by boundless-forest — APPROVED** [review]

**Comment by zhouyun-zoe** [comment]
> Merge firstly, then review from website url. This way ok?

It's not OK. Actually I already check in my `localhost`.

**Pull request merged**